docs/DEVELOPMENT.md

```diff
@@ -48,7 +48,7 @@ make docker-push
 
 ## Building Operator bundle image
 ```
-make bundle
+make bundle CHANNELS=candidate
```

Comment:
candidate is also what we use in our example subscription in config/samples/deploy, so I think this is a good change